File: Modul 3/PRAK304/PRAK304-2310817310011-MuhammadRizky.c
#include <stdio.h>

int main(){
    int angkaInput;
    scanf("%d", &angkaInput);

    if (angkaInput < 0 || angkaInput > 99){
        printf("Anda Menginput Melebihi Limit Bilangan");
    } else if (angkaInput > 0 && angkaInput < 10){
        printf("Satuan\n");
    } else if (angkaInput >= 11 && angkaInput < 20){
        printf("Belasan\n");
    } else if (angkaInput >= 20 && angkaInput < 99){
        printf("Puluhan\n");
    } else if (angkaInput == 10){
        printf("Puluhan\n");
    }
    else if (angkaInput == 0){
        printf("Nol\n");
    }
    
    return 0;
}

File: Modul 3/PRAK304/PRAK304-2310817310011-MuhammadRizky.py
angkaInput = int(input())

if (angkaInput < 0 or angkaInput > 99):
    print("Anda Menginput Melebihi Limit Bilangan")
elif (angkaInput == 0):
    print("Nol")
elif (angkaInput > 0 and angkaInput < 10):
    print("Satuan")
elif (angkaInput > 10 and angkaInput < 20):
    print("Belasan")
elif (angkaInput >= 20 and angkaInput < 99):
    print("Puluhan")
elif (angkaInput == 10):
    print("Puluhan")